include/bluetooth/l2cap.h

```diff
@@ -26,21 +26,61 @@
 extern "C" {
 #endif
 
-/** L2CAP header size, used for buffer size calculations */
+/** L2CAP PDU header size, used for buffer size calculations */
 #define BT_L2CAP_HDR_SIZE               4
 
+/** Maximum Transmission Unit (MTU) for an outgoing L2CAP PDU. */
+#define BT_L2CAP_TX_MTU (CONFIG_BT_L2CAP_TX_MTU)
+
+/** Maximum Transmission Unit (MTU) for an incoming L2CAP PDU. */
+#define BT_L2CAP_RX_MTU (CONFIG_BT_BUF_ACL_RX_SIZE - BT_L2CAP_HDR_SIZE)
```

Comment:
It's a bit early in the morning still, but I can't really wrap my head around why TX MTU simply depends on a separate Kconfig, but RX MTU depends on another Kconfig. 

Can you elaborate a bit?

Reply:
@cvinayak

Reply:
TX MTU is something an application can configure and agrees to use when sending data. Where as Rx MTU has to be derived from the maximum that can arrive over ACL connection with a peer. Hence, here the Rx MTU is calculated from the configured Rx buffer sizes.